doc/INSTALL-Win64.txt

```diff
@@ -0,0 +1,178 @@
+	Install in default location with default options.
+
+- GNU Sed for Win32.
+	Optional for translation file cleanup
+	Get it from http://gnuwin32.sourceforge.net/packages/sed.htm
+	Install in default location with default options.
+
+-------------------------------------------------------------------------------
+Compilation
+-------------------------------------------------------------------------------
+
+- Get the KVIrc source code
+	Using GIT bash:
+	 cd /c/kvirccomp/
+	 git clone https://github.com/kvirc/kvirc.git
+
+- Compile KVIrc
+	Open a Visual Studio x64 Native command prompt. Before starting, we have to set some environment variables.
+	Replace with your respective versions and install paths. The Perl and Python paths might already be set.
+	 set PATH=%PATH%;C:\kvirccomp\zlib-1.2.8;C:\Qt\5.4\msvc2013_64\bin;C:\Perl64\bin;C:\Perl64\site\bin;C:\Python27;C:\Program Files (x86)\GnuWin32\bin
+	 set CMAKE_INCLUDE_PATH=%CMAKE_INCLUDE_PATH%;C:\kvirccomp\openssl-1.0.2d\include;C:\kvirccomp\zlib-1.2.8
+	 set CMAKE_LIBRARY_PATH=%CMAKE_LIBRARY_PATH%;C:\kvirccomp\openssl-1.0.2d\out32dll;C:\kvirccomp\zlib-1.2.8;C:\Qt\5.4\msvc2013_64\lib
+
+	If you compiled OpenSSL yourself and want to use this during compilation,
+	and have the package from SLProWeb installed in it's default location,
+	you need to rename the SLProWeb installationpath first.
+	CMake will look at those default paths first.
+
+	Next, go to the KVIrc source dir and create a build directory and its Makefiles using cmake and compile.
+	We use the "release" build-type as "debug" seems to have incompatibilities between /O2 and /RTC usage
+	If you want spellchecking support, add these flags to the cmake command:
+		"-DEnchant_FOUND=1 -DEnchant_INCLUDE_DIRS=c:\kvirccomp\enchant-inst\include\enchant;c:\kvirccomp\enchant-inst\include\glib-2.0 -DEnchant_LDFLAGS=c:\kvirccomp\enchant-inst\lib\libenchant.dll.a"
+	If you don't want Perl scripting support, add this flag to the cmake command:
+		"-DWANT_PERL=0"
+	If you don't want the Phonon audio backend and want to ignore the warning, add this flag to the cmake command:
+		"-DWANT_PHONON=0"
+	 cd C:\kvirccomp\KVIrc
+	 md build
+	 cd build
+	 cmake .. -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=release -DWANT_KDE=0
+	 nmake
+	 nmake install
+ 
+	Your binaries are now located at C:\kvirccomp\kvirc\build\release.
+	But you can’t start kvirc.exe as there're some libraries missing.
+	You can either copy these to the system dir, add their locations to %PATH% or copy them to the KVIrc-release dir.
+	The latter is preferred for portability reasons.
+	 robocopy C:\Qt\5.4\msvc2013_64\bin C:\kvirccomp\kvirc\build\release\ Qt5Network.dll Qt5Core.dll Qt5Widgets.dll Qt5Gui.dll Qt5WebKitWidgets.dll Qt5WebKit.dll Qt5Sensors.dll Qt5Positioning.dll Qt5Quick.dll Qt5Qml.dll Qt5Multimedia.dll Qt5WebChannel.dll Qt5Sql.dll Qt5MultimediaWidgets.dll Qt5OpenGL.dll Qt5PrintSupport.dll Qt5WinExtras.dll icuin53.dll icuuc53.dll icudt53.dll
```

Comment:
If you use Qt5, it has a special program `windeployqt` which looks at kvirc.exe and copies required .dll files (including plugins).
It's used in AppVeyor build.